src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -183,6 +183,11 @@ internal static class LoggerExtension
             new EventId(335, nameof(MinimumBundleVersionNotSatisfied)),
             "Referenced bundle {bundleId} of version {bundleVersion} does not meet the required minimum version of {minimumVersion}. Update your extension bundle reference in host.json to reference {minimumVersion2} or later.");
 
+        private static readonly Action<ILogger, int, Exception> _functionMetadataProviderFunctionsFoundCustomProvider =
```

Comment:
With the naming above, we can reuse the Provider parameterized messages as is - we don't need a separate one for custom.